client/src/frontend/components/journal/VolumeDetail/index.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import PropTypes from "prop-types";
+import EntityGroup from "global/components/composed/EntityGroup";
+import ThumbnailGrid from "global/components/composed/ThumbnailGrid";
+import EntityThumbnail from "global/components/atomic/EntityThumbnail";
+import ContentBlockList from "frontend/components/content-block-list/List";
+import { fixtures } from "helpers/storybook/exports";
+import * as Styled from "./styles";
+
+// TODO: update once API is in place
+// Current component expects that contentBlock data matches that on projects
+function getIssues(journal) {
+  const sampleTexts = fixtures
+    .collectionFactory("text", 4)
+    .map((text, index) => {
+      return {
+        ...text.data,
+        attributes: {
+          ...text.attributes,
+          position: index + 1
+        }
+      };
+    });
+  return fixtures.collectionFactory("issue", 2).map(issue => {
+    return {
+      ...issue.data,
+      relationships: {
+        texts: sampleTexts,
+        textCategories: [],
+        contentBlocks: [
+          {
+            id: "cf56452c-973f-4244-8b5d-d11dc8c24bfc",
+            type: "textsBlocks",
+            attributes: {
+              type: "Content::TextsBlock",
+              position: 1,
+              visible: true,
+              access: "always",
+              renderable: true,
+              showAuthors: true,
+              showDescriptions: false,
+              showSubtitles: true,
+              showCovers: true,
+              showDates: true,
+              showCategoryLabels: true,
+              title: null,
+              showUncategorized: true
+            },
+            relationships: {
+              project: {
+                data: {
+                  id: "738b2a20-8fd9-4111-93af-f20a8e7fd9cc",
+                  type: "projects"
+                }
+              },
+              includedCategories: []
+            }
+          }
+        ]
+      }
+    };
+  });
+}
+
+function VolumeDetail({ journal, volume }) {
+  const journalTitle = journal.attributes.title;
+  const volumeTitle = volume.attributes.title;
+  const issues = getIssues(journal) || [];
+
+  return (
+    <Styled.Wrapper>
+      <EntityGroup title={`${journalTitle}: ${volumeTitle}`}>
+        {issues.map(issue => (
+          <Styled.IssueWrapper>
+            <ThumbnailGrid key={issue.id} minColumns={4} minItemWidth="210px">
+              {({ stack }) => (
+                <EntityThumbnail entity={issue} stack={stack} key={issue.id} />
+              )}
+            </ThumbnailGrid>
+            <ContentBlockList entity={issue} nested hideHeader />
```

Comment:
Should I be able to see these edit buttons when not logged in? They don't appear at all in design. The icons also aren't in design for volume detail, just issue, but I can see if this was a choice for consistency.

<img width="1102" alt="Screen Shot 2022-01-18 at 11 28 14 AM" src="https://user-images.githubusercontent.com/32903719/150005388-42b4312d-b60e-4bd9-9af2-e74e129fb379.png">

Reply:
These aren't edit buttons; they're supposed to show highlight and annotation counts, but the test fixture I used appears to have incomplete attributes. In any case, issues will have the same content blocks as projects, so I'm just rendering those, without making any changes to design.